package.json

```diff
@@ -13,9 +13,10 @@
   "dependencies": {
     "@adobe/asset-compute-commons": "^1.1.2",
     "@adobe/cgroup-metrics": "^3.0.3",
-    "@adobe/httptransfer": "^2.2.0",
+    "@adobe/httptransfer": "^2.7.0",
     "@adobe/metrics-sampler": "^1.0.2",
     "@adobe/node-fetch-retry": "^1.1.1",
+    "@nui/asset-compute-pipeline": "^0.1.4",
```

Comment:
Once asset-compute-pipeline is open sourced sdk will need updates in package.json.